### PR TITLE
Update Vietnamese translation

### PR DESCRIPTION
**Pull request by dnhn**
Improve and provide translation for missing terms.

### CODE REVIEW

File: _data/trans/vi.json [hunk 1]
```diff
@@ -1,20 +1,20 @@
 {
-    "about": "About",
-    "contribute": "Fork on GitHub",
+    "about": "Giới thiệu",
+    "contribute": "Fork trên GitHub",
```
Comment (tupaschoal) [inline _data/trans/vi.json:3]: This became `"contribute": "Contribute on GitHub"` on the source language, does this take it into consideration?

**Comment by Nightfirecat** [comment]
Thanks for your contribution!

File: _data/trans/vi.json [hunk 2]
```diff
@@ -25,9 +25,9 @@
     "mikerogers": "Mike Rogers",
     "name": "Tiếng Việt",
     "noinfo": "Không có thông tin",
-    "noresults": "Can't find what you're looking for?",
-    "noresultshelp": "Help make JustDeleteMe better",
-    "popular": "Popular",
+    "noresults": "Không tìm thấy thông tin bạn cần?",
+    "noresultshelp": "Hãy làm cho JustDeleteMe trở nên tốt hơn",
+    "popular": "Phổ biến",
     "pullrequest": "Gia nhập dự án trên GitHub",
```
Comment (tupaschoal) [inline _data/trans/vi.json:31]: This became `"pullrequest": "Make a Pull Request on GitHub"` on the source language, can you confirm that it was already updated, please?

**Comment by dnhn** [comment]
@tupaschoal thanks for pointing out, they will be updated in the next PR.

**Pull request merged**